src/megachatapi.h

```diff
@@ -2709,11 +2709,46 @@ class MegaChatApi
     void inviteToChat(MegaChatHandle chatid, MegaChatHandle uh, int privilege, MegaChatRequestListener *listener = NULL);
 
     /**
-     * @brief TODO documentation
+     * @brief Allow a user to add himself to an existing public chat. To do this the public chat
+     * must have a valid public handle.
```

Comment:
Further than to have a valid public handle, I'd say the chatroom must be in preview mode, the result of a previous call to `loadChatLink()` and the public handle contained in the chat-link must be still valid.